Ctor 457 new pack network juniper mseries netconf (#4416)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md

> import Tabs from '@theme/Tabs';

Reviewer: Juniper M-Series SNMP?

Reviewer 2: SAAS-1970

import TabItem from '@theme/TabItem';

## Dépendances du connecteur de supervision

Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Juniper M-Series** 

> depuis la page **Configuration > Gestionnaire de Connecteurs de supervision** :

Reviewer: ```suggestion
depuis la page **Configuration > Connecteurs > Connecteurs de supervision** :
```

> depuis la page **Configuration > Gestionnaire de Connecteurs de supervision** :

Reviewer 2: ```suggestion
depuis la page **Configuration > Connecteurs > Connecteurs de supervision** :
```

</TabItem>
<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">

| Alias                      | Modèle de service                                          | Description                                                                                                                                                 | Découverte |
|:---------------------------|:-----------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------:|

> | Bgp-Peer-Prefix-Statistics | Net-Juniper-Mseries-SNMP-Bgp-Peer-Prefix-Statistics-custom | Contrôle les statistiques des prefix des pairs BGP                                                                                                          |            |

Reviewer 2: ```suggestion
| Bgp-Peer-Prefix-Statistics | Net-Juniper-Mseries-SNMP-Bgp-Peer-Prefix-Statistics-custom | Contrôle les statistiques des préfixes des pairs BGP                                                                                                          |            |
```

</TabItem>
<TabItem value="Rsvp-Session-Status" label="Rsvp-Session-Status">

| Nom    | Unité |
|:-------|:------|
| status | N/A   |

> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.

</TabItem>
<TabItem value="Traffic-*" label="Traffic-*">

> Coming soon

Reviewer 2: ```suggestion
| Nom                                           | Unité |
|:-----------------------------------------------------|:------|
| *interface_name*#status                              | N/A   |
| *interface_name*#interface.traffic.in.bitspersecond  | b/s   |
| *interface_name*#interface.traffic.out.bitspersecond | b/s   |

> Concerne les modèles de service suivants : Traffic-Generic-Id, Traffic-Generic-Name, Traffic-Global
```

### Pack

> La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md). de supervision

Reviewer 2: ```suggestion
La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md) de supervision.
```

### Pack

> La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md) de supervision.

Reviewer: ```suggestion
La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md).
```

> La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md) de supervision.

Reviewer 2: ```suggestion
La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md).
```

1. Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet

> n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer: ```suggestion
n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

> n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer 2: ```suggestion
n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

</Tabs>

2. Quel que soit le type de la licence (*online* ou *offline*), installez le connecteur **Juniper M-Series**

> depuis l'interface web et le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer: ```suggestion
depuis l'interface web et le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

> depuis l'interface web et le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer 2: ```suggestion
depuis l'interface web et le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

3. Appliquez le modèle d'hôte **Net-Juniper-Mseries-SNMP-custom**.

> Si vous utilisez SNMP en version 3, vous devez configurer les paramètres spécifiques associés via la macro **SNMPEXTRAOPTIONS**.

> > Plus d'informations dans la section [Troubleshooting SNMP](../getting-started/how-to-guides/troubleshooting-plugins.md#snmpv3-options-mapping).

Reviewer: ```suggestion
> Plus d'informations dans la section [Troubleshooting SNMP](../getting-started/how-to-guides/troubleshooting-plugins.md#mapping-des-options-snmpv3).
```

> > Plus d'informations dans la section [Troubleshooting SNMP](../getting-started/how-to-guides/troubleshooting-plugins.md#snmpv3-options-mapping).

Reviewer 2: ```suggestion
> Plus d'informations dans la section [Troubleshooting SNMP](../getting-started/how-to-guides/troubleshooting-plugins.md#mapping-des-options-snmpv3).
```

| FILTERLOCALAS  | Filter by local AS (can be regexp)                                                                                                                                                                                                                                                                                                                                                                             |                                                                     |             |
| CRITICALSTATUS | Specify critical threshold (default: <'%\{peer\_status\} =~ /running/ && %\{peer\_state\} !~ /established/'\>). Can use special variables like <%\{peer\_identifier\}\>, <%\{peer\_state\}\>, <%\{peer\_status\}\>, <%\{local\_type\}\>, <%\{local\_ip\}\>, <%\{local\_port\}\>, <%\{local\_as\}\>, <%\{remote\_type\}\>, <%\{remote\_ip\}\>, <%\{remote\_port\}\>, <%\{remote\_as\}\> | %\{peer\_status\} =~ /running/ && %\{peer\_state\} !~ /established/ |             |

> | WARNINGSTATUS  | Specify warning threshold. Can use special variables like <%\{peer\_identifier\}\>, <%\{peer\_state\}\>, <%\{peer\_status\}\>, <%\{local\_type\}\>, <%\{local\_ip\}\>, <%\{local\_port\}\>, <%\{local\_as\}\>, <%\{remote\_type\}\>, <%\{remote\_ip\}\>, <%\{remote\_port\}\>, <%\{remote\_as\}\>                                                                                        |                                                                     |             |

Reviewer 2: ```suggestion
| CRITICALSTATUS | Specify critical threshold (default: '%\{peer_status\} =~ /running/ && %\{peer_state\} !~ /established/'). Can use special variables like %\{peer_identifier\}, %\{peer_state\}, %\{peer_status\}, %\{local_type\}, %\{local_ip\}, %\{local_port\}, %\{local_as\}, %\{remote_type\}, %\{remote_ip\}, %\{remote_port\}, %\{remote_as\} | %\{peer_status\} =~ /running/ && %\{peer_state\} !~ /established/ |             |
| WARNINGSTATUS  | Specify warning threshold. Can use special variables like %\{peer_identifier\}, %\{peer_state\}, %\{peer_status\}, %\{local_type\}, %\{local_ip\}, %\{local_port\}, %\{local_as\}, %\{remote_type\}, %\{remote_ip\}, %\{remote_port\}, %\{remote_as\}                                                                               |                                                                 |             |
```

> | WARNINGSTATUS  | Specify warning threshold. Can use special variables like C\<%\{peer\_identifier\}\>, C\<%\{peer\_state\}\>, C\<%\{peer\_status\}\>, C\<%\{local\_type\}\>, C\<%\{local\_ip\}\>, C\<%\{local\_port\}\>, C\<%\{local\_as\}\>, C\<%\{remote\_type\}\>, C\<%\{remote\_ip\}\>, C\<%\{remote\_port\}\>, C\<%\{remote\_as\}\>                                                                                        |                                                                     |             |

Reviewer: C?

> | WARNINGSTATUS  | Specify warning threshold. Can use special variables like C\<%\{peer\_identifier\}\>, C\<%\{peer\_state\}\>, C\<%\{peer\_status\}\>, C\<%\{local\_type\}\>, C\<%\{local\_ip\}\>, C\<%\{local\_port\}\>, C\<%\{local\_as\}\>, C\<%\{remote\_type\}\>, C\<%\{remote\_ip\}\>, C\<%\{remote\_port\}\>, C\<%\{remote\_as\}\>                                                                                        |                                                                     |             |

Reviewer 2: ```suggestion
| CRITICALSTATUS | Specify critical threshold (default: <'%\{peer\_status\} =~ /running/ && %\{peer\_state\} !~ /established/'\>). Can use special variables like <%\{peer\_identifier\}\>, <%\{peer\_state\}\>, <%\{peer\_status\}\>, <%\{local\_type\}\>, <%\{local\_ip\}\>, <%\{local\_port\}\>, <%\{local\_as\}\>, <%\{remote\_type\}\>, <%\{remote\_ip\}\>, <%\{remote\_port\}\>, <%\{remote\_as\}\> | %\{peer\_status\} =~ /running/ && %\{peer\_state\} !~ /established/ |             |
| WARNINGSTATUS  | Specify warning threshold. Can use special variables like <%\{peer\_identifier\}\>, <%\{peer\_state\}\>, <%\{peer\_status\}\>, <%\{local\_type\}\>, <%\{local\_ip\}\>, <%\{local\_port\}\>, <%\{local\_as\}\>, <%\{remote\_type\}\>, <%\{remote\_ip\}\>, <%\{remote\_port\}\>, <%\{remote\_as\}\>                                                                                        |                                                                     |             |
```

| CRITICAL     | Critical threshold                                                                                                                                                                              | 95                     |             |
| WARNING      | Warning threshold                                                                                                                                                                               | 90                     |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                                                              | --verbose              |             |

</TabItem>
<TabItem value="Hardware" label="Hardware">

| Macro        | Description                                                                                        | Valeur par défaut | Obligatoire |
|:-------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | COMPONENT    | Which component to check (default: C\<'.*'\>). Can be: `fru`, `operating`, `alarm`                 | .*                |             |

Reviewer: C?

> | COMPONENT    | Which component to check (default: C\<'.*'\>). Can be: `fru`, `operating`, `alarm`                 | .*                |             |

Reviewer 2: ```suggestion
| COMPONENT    | Which component to check (default: \<'.*'\>). Can be: `fru`, `operating`, `alarm`                 | .*                |             |
```

| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). | --verbose         |             |

</TabItem>
<TabItem value="Ldp-Session-Status" label="Ldp-Session-Status">

| Macro              | Description                                                                                                                                             | Valeur par défaut            | Obligatoire |
|:-------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------|:-----------:|
| FILTERENTITY       | Threshold                                                                                                                                               |                              |             |

> | FILTERPEER         | Threshold                                                                                                                                               |                              |             |

Reviewer 2: ```suggestion
| FILTERENTITY       | Filter entities                                                                                                                         |                            |             |
| FILTERPEER         | Filter peers                                                                                                                         |                            |             |
```

| FILTERFROM              | Filter LSPs                                                                                                                                |                   |             |
| FILTERTO                | Filter LSPs                                                                                                                                |                   |             |
| WARNINGLASTTRANSITION   | Threshold                                                                                                                                                |                     |             |
| CRITICALLASTTRANSITION  | Threshold                                                                                                                                                |                     |             |
| CRITICALSTATUS          | Define the conditions to match for the status to be CRITICAL. You can use the following variables: <%\{state\}\> | %\{state\} !~ /up/i |             |

> | WARNINGSTATUS           | Define the conditions to match for the status to be WARNING. You can use the following variables: <%\{state\}\>                        |                     |             |

Reviewer 2: ```suggestion
| CRITICALSTATUS          | Define the conditions to match for the status to be CRITICAL. You can use the following variables: %\{state\} | %\{state\} !~ /up/i |             |
| WARNINGSTATUS           | Define the conditions to match for the status to be WARNING. You can use the following variables: %\{state\}                        |                     |             |
```

> | WARNINGSTATUS           | Define the conditions to match for the status to be WARNING (default: `''`). You can use the following variables: <%\{state\}\>                        |                     |             |

Reviewer 2: ```suggestion
| CRITICALSTATUS          | Define the conditions to match for the status to be CRITICAL. You can use the following variables: <%\{state\}\> | %\{state\} !~ /up/i |             |
| WARNINGSTATUS           | Define the conditions to match for the status to be WARNING. You can use the following variables: <%\{state\}\>                        |                     |             |
```

> | FILTERTO                | Threshold                                                                                                                                                |                     |             |

Reviewer 2: ```suggestion
| FILTERNAME              | Filter LSPs                                                                                                                                |                   |             |
| FILTERFROM              | Filter LSPs                                                                                                                                |                   |             |
| FILTERTO                | Filter LSPs                                                                                                                                |                   |             |
```

> | CRITICALLASTTRANSITION  | Threshold                                                                                                                                                |                     |             |

Reviewer: filter/threshold?

> | CRITICALSTATUS          | Define the conditions to match for the status to be CRITICAL (default: C\<'%\{state\} !~ /up/i'\>). You can use the following variables: C\<%\{state\}\> | %\{state\} !~ /up/i |             |

Reviewer: C?

> | WARNINGSTATUS           | Define the conditions to match for the status to be WARNING (default: `''`). You can use the following variables: C\<%\{state\}\>                        |                     |             |

Reviewer: C?

> | WARNINGSTATUS           | Define the conditions to match for the status to be WARNING (default: `''`). You can use the following variables: C\<%\{state\}\>                        |                     |             |

Reviewer 2: ```suggestion
| CRITICALSTATUS          | Define the conditions to match for the status to be CRITICAL (default: <'%\{state\} !~ /up/i'\>). You can use the following variables: <%\{state\}\> | %\{state\} !~ /up/i |             |
| WARNINGSTATUS           | Define the conditions to match for the status to be WARNING (default: `''`). You can use the following variables: <%\{state\}\>                        |                     |             |
```

| EXTRAOPTIONS      | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). | --verbose         |             |

</TabItem>
<TabItem value="Rsvp-Session-Status" label="Rsvp-Session-Status">

| Macro          | Description                                                                                                                                    | Valeur par défaut   | Obligatoire |
|:---------------|:-----------------------------------------------------------------------------------------------------------------------------------------------|:--------------------|:-----------:|
| FILTERNAME     | Threshold                                                                                                                                      |                     |             |
| FILTERFROM     | Threshold                                                                                                                                      |                     |             |

> | FILTERTO       | Threshold                                                                                                                                      |                     |             |

Reviewer: filter/threshold?

> | FILTERTO       | Threshold                                                                                                                                      |                     |             |

Reviewer 2: ```suggestion
| FILTERNAME     | Filter sessions                                                                                                                            |                   |             |
| FILTERFROM     | Filter sessions                                                                                                                            |                   |             |
| FILTERTO       | Filter sessions                                                                                                                            |                   |             |
```

</TabItem>
<TabItem value="Traffic-Generic-Id" label="Traffic-Generic-Id">

| Macro        | Description                                                                                        | Valeur par défaut | Obligatoire |
|:-------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| INTERFACEID  | Set the interface (number expected) example: 1,2,... (empty means 'check all interfaces')          |                   |             |
| CRITICALIN   | Set critical threshold for in traffic                                                              | 90                |             |
| WARNINGIN    | Set warning threshold for in traffic                                                               | 80                |             |
| CRITICALOUT  | Set critical threshold for out traffic                                                             | 90                |             |

> | WARNINGOUT   | Set warning threshold for out traffic                                                              | 80                |             |

Reviewer: why so much difference between the old and the new version?

Reviewer 2: because old one was wrong

| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). |                   |             |

</TabItem>
<TabItem value="Traffic-Generic-Name" label="Traffic-Generic-Name">

| Macro         | Description                                                                                        | Valeur par défaut | Obligatoire |
|:--------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| INTERFACENAME | Set the interface (number expected) example: 1,2,... (empty means 'check all interfaces')          |                   |             |
| CRITICALIN    | Set critical threshold for in traffic                                                              | 90                |             |
| WARNINGIN     | Set warning threshold for in traffic                                                               | 80                |             |
| CRITICALOUT   | Set critical threshold for out traffic                                                             | 90                |             |

> | WARNINGOUT    | Set warning threshold for out traffic                                                              | 80                |             |

Reviewer: same

Reviewer 2: same

| EXTRAOPTIONS  | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). |                   |             |

</TabItem>
<TabItem value="Traffic-Global" label="Traffic-Global">

| Macro          | Description                                                                                                                                                                                                                                              | Valeur par défaut | Obligatoire |
|:---------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| FILTER         | Set the interface (number expected) example: 1,2,... (empty means 'check all interfaces')                                                                                                                                                                | .*                |             |

> | WARNINGIN      | Set warning threshold for in traffic                                                                                                                                                                                                                     | 80                |             |

Reviewer: same

Reviewer 2: same

| CRITICALIN     | Set critical threshold for in traffic                                                                                                                                                                                                                    | 90                |             |
| WARNINGOUT     | Set warning threshold for out traffic                                                                                                                                                                                                                    | 80                |             |
| CRITICALOUT    | Set critical threshold for out traffic                                                                                                                                                                                                                   | 90                |             |

> | CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: C\<'%\{admstatus\} eq "up" and %\{opstatus\} ne "up"'\>). You can use the following variables: C\<%\{admstatus\}\>, C\<%\{opstatus\}\>, C\<%\{duplexstatus\}\>, C\<%\{display\}\> |                   |             |

Reviewer: C?

> | CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: C\<'%\{admstatus\} eq "up" and %\{opstatus\} ne "up"'\>). You can use the following variables: C\<%\{admstatus\}\>, C\<%\{opstatus\}\>, C\<%\{duplexstatus\}\>, C\<%\{display\}\> |                   |             |

Reviewer 2: ```suggestion
| CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: '%\{admstatus\} eq "up" and %\{opstatus\} ne "up"'). You can use the following variables: %\{admstatus\}, %\{opstatus\}, %\{duplexstatus\}, %\{display\} |                   |             |
```
